mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
+  try {
+    const order = await Order.findByIdAndRemove(req.params.orderId);
+    if (!order) {
+      return res.status(404).json({ message: "Order does not exist" });
+    }
+    res.status(200).json({ message: "Order deleted successfully" });
```

Comment:
The response object should be consistent throughout the application. In this file, other success responses are sent as `{ error: null, data: ... }`. This response should be changed to follow the same pattern. It should be `res.json({ error: null, message: "Order deleted successfully" });`.